test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -83,6 +83,68 @@ public void CreatePackageWithEmptyFoldersForV3Folders()
             }
         }
 
+        [Fact]
+        public void CreatePackageWithFiles()
```

Comment:
Don't other tests create packages with files?

I don't know if it's in our coding guidelines document, but a common naming pattern we use for unit tests is `<MethodBeingTested>_<Scenario>_<ExpectedResult>`. For example: https://github.com/NuGet/NuGet.Client/blob/1dd5351295a5a0d092abcd2bd82ccbd730682dbe/test/NuGet.Core.Tests/NuGet.Packaging.Test/InvalidFrameworkFolderRuleTests.cs#L19

I see that none of the other tests in this file follow that pattern, but this `CreatePackageWithFiles` name is too generic and I can't guess what it's supposed to test.